library/swift/EngineBuilder.swift

```diff
@@ -169,6 +170,17 @@ public final class EngineBuilder: NSObject {
     return self
   }
 
+  /// Set a closure to be called when the engine's logger logs.
+  ///
+  /// - parameter closure: The closure to be called.
+  ///
+  /// - returns: This builder.
+  @discardableResult
+  public func setOnEngineLog(closure: @escaping (String) -> Void) -> EngineBuilder {
```

Comment:
Mapping in Objective-C I can kind of take it or leave it with my above comment. But here, I really do object to this naming scheme. I feel pretty strongly we should expose the public interface as "Logger" or some variant thereof.